gradle-plugin-release-plugin/plugin/src/test/groovy/com/novoda/releaseplugin/PrintChangelogIntegrationTest.groovy

```diff
@@ -0,0 +1,113 @@
+package com.novoda.releaseplugin
+
+import com.google.common.io.Resources
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+
+import static com.google.common.truth.Truth.assertThat
+
+class PrintChangelogIntegrationTest {
+
+    @Rule
+    public final TemporaryFolder testProjectDir = new TemporaryFolder()
+    private File buildFile
+
+    @Before
+    void setup() throws IOException {
+        buildFile = testProjectDir.newFile("build.gradle")
+    }
+
+    @Test
+    void shouldPrintChangelog() throws IOException {
+        configureExtensionWith('0.1', Resources.getResource("CHANGELOG.md"))
+
+        BuildResult build = buildWithArgs('printReleaseChangelog')
+
+        assertThat(build.output).contains("- Initial release.")
+    }
+
+    @Test
+    void shouldNotRunParseChangelogTwiceWhenVersionAndChangelogAreSame() {
```

Comment:
I think groovy would allow spaces like `'should not run parse changelog twice'` here, similarly to kotlin.